modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -102,9 +102,43 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.6.1 also includes the following addition:
 
-=== New `pad_empty_with_br` option that can be set to `true` to pad empty block elements with `<br>` tag instead of nbsp character
+=== New filtering option: `pad_empty_with_br`
 //#TINY-9861
 
+{productname} 6.6.1 includes a further content filtering option, `pad_empty_with_br`.
+
+Content filtering options change the way {productname} manages input and output.
+
+The `pad_empty_with_br` option, when set to `true`, places a `+<br>+` tag within empty block elements rather than the default non-breaking space entity, `+&nbsp;+`.
+
+That is, when a {productname} instance includes the following setting
+
+[source,js]
+----
+pad_empty_with_br: true,
+----
+ 
+this input
+
+[source,html]
+----
+<p>&nbsp;</p>
+----
+
+is automatically stored and rendered as
```

Comment:
Here is not well clear what input and output mean. This option relates to how the content is serialized. So input here is what is actually in the browser DOM. For example, it is an empty paragraph `<p></p>`. The output is the result of `editor.getContent()` or what the user sees in the document source code viewer. If the option is set to `true` the empty paragraph will be converted to `<p><br></p>`, otherwise to `<p>&nbsp;</p>`.

Reply:
To quote my commit message:

> General I/O framing replaced with explicit notes regarding the TinyMCE instance automatically storing empty block elements with embedded break tags.